controllers/current.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/current", async (req, res) => {
+  const { id, email } = req.user;
+  try {
+    res.json({
+      status: "success",
+      code: 200,
+      data: { id, email },
+      message: "User data",
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad Request",
+    });
+  }
```

Comment:
```suggestion
    res.json({
      status: "success",
      code: 200,
      data: { id, email },
      message: "User data",
    });
```
Nie robisz try na res.

Reply:
Doczytaj kiedy się używa try catch. Generalnie zawsze kiedy jest async i w niektórych sytuacjach obsługując synchroniczne funkcje